.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
.DS_Store can be ignore
package-lock should not always be ignored, as this can lock down versions and unwanted changes from breaking udpates in collaborative teams, take a look at https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json